source/cost-benefits-of-pay.html.erb

```diff
@@ -1,59 +1,13 @@
 ---
-title: The cost benefits of using GOV.UK Pay
+title: Cost benefits of Pay
```

Comment:
Let's keep the old text to match the h1

Reply:
I simplified these to make active page styles work, we were previously comparing the page url to the link in the markup but this doesn't work now that relative links are switched on.

Reply:
These are page titles, they don't need to match the H1 content word for word (and in my view, should be more concise)